docker/bookkeeper/entrypoint.sh

```diff
@@ -9,6 +9,9 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 
+ZK_HOME=/opt/zk
```

Comment:
Maybe this discussion doesn't belong here, but I wonder how this is going to work with upgrades. If we use always the same directory, then do we overwrite the content? Or do we assume for now no upgrades?

Reply:
In almost all use cases, `entrypoint.sh` will be run inside a docker container. I am assuming when we upgrade versions, we will just release a newer version of the docker container.